### PR TITLE
ARC-326: deployment to prod

### PR DESCRIPTION
**Pull request by thombergs**

### CODE REVIEW

File: github-for-jira.sd.yml [hunk 1]
```diff
@@ -12,6 +12,15 @@ compose:
     ports:
       - 8080:8080
 
+computeClassification:
+  dataType:
```
Comment (thombergs) [inline github-for-jira.sd.yml:16]: added all the data classification labels because that is a requirement for prod

File: github-for-jira.sd.yml [hunk 2]
```diff
@@ -101,20 +118,18 @@ environmentOverrides:
         INSTANCE_NAME: github-for-jira-dev
         NODE_OPTIONS: "--no-deprecation"
         LOG_LEVEL: debug
-        # * Commented out env vars are to be saved in secrets
-        # WEBHOOK_SECRET: for signed signature when syncing an org
-        # MICROS_ENV: micros environment for deployment
-        # MICROS_ENVTYPE: the type of micros environment for deployment
+        SENTRY_ENVIRONMENT: ddev
+        APP_ID: '124403'
+        GITHUB_CLIENT_ID: Iv1.600bf90a20f1ab18
```
Comment (thombergs) [inline github-for-jira.sd.yml:123]: Moved all non-secret environment variables into the service descriptor for better visibility.

File: github-for-jira.sd.yml [hunk 3]
```diff
@@ -129,8 +144,12 @@ environmentOverrides:
   prod:
     config:
       environmentVariables:
-        APP_URL: https://github-for-jira.prod.services.atlassian.com
+        APP_URL: https://github-for-jira.services.atlassian.com
+        WEBHOOK_PROXY_URL: https://github-for-jira.services.atlassian.com/github/events
         INSTANCE_NAME: github-for-jira-production
+        SENTRY_ENVIRONMENT: production
+        APP_ID: '125082'
+        GITHUB_CLIENT_ID: Iv1.234a3f06f2b64fbe
```
Comment (thombergs) [inline github-for-jira.sd.yml:152]: This is configured against a new GitHub app I created as interim for until we're moving to the "real" GitHub app with the migration.

**Review by rachellerathbone — APPROVED** [review]

File: github-for-jira.sd.yml (hunk 1 above)
Comment (KAbakumovAtl) [reply github-for-jira.sd.yml:16]: Thanks a lot tom! Surprisingly it doesn't seem to be a CREDO task. Is it a Service Linter requirement?

**Pull request merged**